keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlVerificationRepository.kt

```diff
@@ -193,9 +192,9 @@ class SqlVerificationRepository(
       .and(ENVIRONMENT.NAME.eq(environment.name))
       .and(ENVIRONMENT.DELIVERY_CONFIG_UID.eq(DELIVERY_CONFIG.UID))
 
-  private val DeliveryArtifact.uid: Select<Record1<String>>
+  private val VerificationContext.artifactUid: Select<Record1<String>>
     get() = select(DELIVERY_ARTIFACT.UID)
       .from(DELIVERY_ARTIFACT)
-      .where(DELIVERY_ARTIFACT.NAME.eq(name))
-      .and(DELIVERY_ARTIFACT.TYPE.eq(type))
+      .where(DELIVERY_ARTIFACT.DELIVERY_CONFIG_NAME.eq(deliveryConfig.name))
```

Comment:
We should really be using the delivery config uid in these tables consistently. Maybe not worth doing now, but thought I'd mention it while you're at it.

Reply:
not sure what you mean, that's the thing we're trying to select here

Reply:
Yeah, I think @emjburns had some reason why there's not a direct foreign key on this table. Can't remember now.

Reply:
My understanding was that that was only for `artifact_versions`, since those can be reused across multiple `deliver_artifact` records, but a `delivery_artifact` record is always tied to a specific `delivery_config` (hence the "soft" back reference here based on the name).